src/sentry/rules/conditions/event_frequency.py

```diff
@@ -142,7 +143,9 @@ def passes(self, event: GroupEvent, state: EventState) -> bool:
         # TODO(mgaeta): Bug: Rule is optional.
         current_value = self.get_rate(event, interval, self.rule.environment_id)  # type: ignore[arg-type, union-attr]
         logging.info("event_frequency_rule current: %s, threshold: %s", current_value, value)
-        return current_value > value
+        if current_value:
+            return current_value > value
+        return False
```

Comment:
Would it be simpler to put the try/catch here around `self.get_rate`? Then maybe we don't need to change function signatures etc